translations/locales/hi/translations.json

```diff
@@ -0,0 +1,608 @@
+{
+    "Nav": {
+      "File": {
+        "Title": "फाइल",
+        "New": "नई",
+        "Share": "शेयर",
+        "Duplicate": "डुप्लिकेट",
+        "Open": "ओपन",
+        "Download": "डाउनलोड",
+        "AddToCollection": "संग्रह में जोड़ें",
+        "Examples": "उदाहरण"
+      },
+      "Edit": {
+        "Title": "संपादित करे",
+        "TidyCode": "साफ कोड",
+        "Find": "खोज",
+        "FindNext": "अगला खोजें",
+        "FindPrevious": "पिछला खोजें",  
+        "Replace": "बदली करें"
+      },
+      "Sketch": {
+        "Title": "स्केच",
+        "AddFile": "फाइल जोड़ें",
+        "AddFolder": "फोल्डर जोड़ें",
+        "Run": "चलाएं",
+        "Stop": "बंद करें"
+      },
+      "Help": {
+        "Title": "मदद",
+        "KeyboardShortcuts": "कीबोर्ड शॉर्टकट",
+        "Reference": "रिफरेन्स",
+        "About": "के बारे में"
+      },
+      "Lang": "भाषा",
+      "BackEditor": "एडिटर पर वापस जाएं",
+      "WarningUnsavedChanges": "क्या आप इस पेज को छोड़ना चाहते हैं? आपके पास अनसेव्ड परिवर्तन हैं।",
+      "Login": "लॉग इन",
+      "LoginOr": "या",
+      "SignUp": "साइन अप",
+      "Auth": {
+        "Welcome": "स्वागत है",
+        "Hello": "नमस्कार",
+        "MyAccount": "मेरा अकाउंट",
+        "My": "मेरा",
+        "MySketches": "मेरे स्केच",
+        "MyCollections": "मेरे संग्रह",
+        "Asset": "ऐसेट",
+        "MyAssets": "मेरे ऐसेट",
+        "LogOut": "लॉग आउट"
+      }
+    },
+    "CodemirrorFindAndReplace": {
+      "ToggleReplace": "टॉगल बदली करें",
+      "Find": "खोज",
+      "FindPlaceholder": "फ़ाइलों में खोजें",
+      "Replace": "बदली करें",
+      "ReplaceAll": "सबको बदली करें",
+      "ReplacePlaceholder": "बदलने के लिए पाठ",
+      "Regex": "रेगुलर एक्सप्रेशन",
+      "CaseSensitive": "केस सेंसिटिव",
+      "WholeWords": "संपूर्ण शब्द",
+      "Previous": "पिछला",
+      "Next": "अगला",
+      "NoResults": "कोई परिणाम नहीं",
+      "Close": "बंद करे"
+    },
+    "LoginForm": {
+      "UsernameOrEmail": "ईमेल या यूजरनेम",
+      "UsernameOrEmailARIA": "ईमेल या यूजरनेम",
+      "Password": "पासवर्ड",
+      "PasswordARIA": "पासवर्ड",
+      "Submit": "लॉग इन"
+    },
+    "LoginView": {
+      "Title": "p5.js वेब एडिटर | लॉग इन",
+      "Login": "लॉग इन",
+      "LoginOr": "या",
+      "SignUp": "साइन अप",
+      "Email": "ईमेल",
+      "Username": "यूजरनेम",
+      "DontHaveAccount": "अकाउंट नहीं है? ",
+      "ForgotPassword": "पासवर्ड भूल गए? ",
+      "ResetPassword": "पासवर्ड रीसेट करें"
+    },
+    "SocialAuthButton": {
+      "Connect": "कनेक्ट {{serviceauth}} अकाउंट",
+      "Unlink": "अनलिंक {{serviceauth}} अकाउंट",
+      "Login": "{{serviceauth}} से लोगिन करें",
+      "LogoARIA": "{{serviceauth}} प्रतीक चिन्ह"
+    },
+    "About": {
+      "Title": "के बारे में",
+      "TitleHelmet": "p5.js वेब एडिटर | के बारे में",
+      "Contribute": "योगदान",
+      "NewP5": "p5.js पर नये?",
+      "Report": "बग रिपोर्ट",
+      "Learn": "सीखें",
+      "Resources": "रीसॉर्सेज़",
```

Comment:
`"Resources": "साधन"`

Reply:
Phonetic translation for the technical term "Resources" was not matching. Here, a translation(like in the case of the word "Example") would be more appropriate.

Reply:
I was looking for a better word for this translation. Thanks.